_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -20,23 +20,24 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any HTML element that is [visible][] and has one of the following CSS properties set by the `style` attribute:
+This rule applies to any HTML element that is [visible][] and has a [declared value][] for any of the following CSS properties:
 
-- [word-spacing][]
-- [letter-spacing][]
-- [line-height][]
+- [word-spacing][],
+- [letter-spacing][],
+- [line-height][];
+  which are set by the `style` attribute, and have the [important flag][].
```

Comment:
Small problem now:
if only "word spacing" has `!important` then the target is applicable and by Expectation 2 needs "letter spacing" to be large enough (even though it doesn't have `!important`).

That's also why I was thinking maybe 3 small rules is better to streamline both applicability and expectations.

Reply:
@Jym77 I think this can be solved by a play with words in the expectation. How about the below:

`The test target's computed word spacing if having the important flag is at least 0.16 times it's computed font size`.

It makes sense to keep the rules related to text spacing together.

Reply:
Yes, I'm having mixed feeling on my own suggestion to split…
On one hand, I do agree that that keeping these together makes sense as they are very similar and closely related.
On the other hand, if the rule ends up with an applicability of "A or B or C" and three expectations of "If A, then A' ", "If B, then B' ", "If C, then C' ", this sounds so much like 3 rules artificially glued together…

Also, with expectations like "if the important flag is set on XXX, then the value of XXX is at least xx", there is no real need to have the important flag in the applicability.

Which maybe was indeed not needed (my bad). I feel the new wording you're suggesting for expectations is easier to understand than the previous one… Maybe we can go even further:

Applicability: any element where one of this properties is set.
Expectation 1: For each test target, either word spacing is not set, or it does not have the important flag, or its value is at least xx
Expectation : …

WDYT?

Reply:
@Jym77 Yeah I was thinking the same, bring back important to the expectation. We are going back to where the rule was :). But we are getting better.

Reply:
👍

Reply:
@Jym77 pushed an update. Will resolve this conversation & await review.